test/elixir_internal_certificate_web/controllers/api/v1/user_session_controller_test.exs

```diff
@@ -0,0 +1,91 @@
+defmodule ElixirInternalCertificateWeb.Api.V1.UserSessionControllerTest do
+  use ElixirInternalCertificateWeb.ConnCase, async: true
+
+  alias ElixirInternalCertificate.Account.Guardian
+
+  describe "POST create/2" do
+    test "given valid user credentials, returns access token", %{conn: conn} do
+      email = "test@test.com"
+      insert(:user, email: email)
+
+      conn =
+        post(conn, Routes.api_user_session_path(conn, :create), %{
+          email: email,
+          password: valid_user_password()
+        })
+
+      assert %{
+               "data" => %{
+                 "attributes" => %{
+                   "email" => ^email,
+                   "token" => _
+                 },
+                 "id" => _,
+                 "relationships" => %{},
+                 "type" => "user_session"
+               },
+               "included" => []
+             } = json_response(conn, 200)
+    end
+
+    test "given invalid user credentials, returns error response", %{conn: conn} do
+      email = "test@test.com"
+      insert(:user, email: email)
+
+      conn =
+        post(conn, Routes.api_user_session_path(conn, :create), %{
+          email: "test@gmail.com",
+          password: "invalid_password"
+        })
+
+      assert json_response(conn, 401) == %{
+               "errors" => [
+                 %{
+                   "code" => "unauthorized",
+                   "detail" => "Invalid email or password"
+                 }
+               ]
+             }
+    end
+
+    test "given not correct input attributes, returns error response", %{conn: conn} do
```

Comment:
how about
```suggestion
    test "given invalid request params, returns error response", %{conn: conn} do
```

Reply:
Updated in [9af1a49](https://github.com/nvminhtue/elixir_internal_certificate/pull/43/commits/9af1a4946750fcd6f06a7f549f31f502914fc951)